src/components/Auth.jsx

```diff
@@ -36,71 +41,43 @@ user is redirected to the URL they attempted to login from initially.
 export class Auth extends React.Component {
```

Comment:
Should `Auth` still be exported anymore, or should we only exported it after connecting it?